torch/nn/functional.py

```diff
@@ -245,8 +245,9 @@
         averaging calculation. Default: ``True``
 
 Examples::
+
     >>> # pool of square window of size=3, stride=2
-    >>> input = torch.tensor([[[1,2,3,4,5,6,7]]])
+    >>> input = torch.tensor([[[1.,2,3,4,5,6,7]]])
```

Comment:
Ah thanks for spotting this! Could you maybe change this line to 
```py
>>> input = torch.tensor([[[1, 2, 3, 4, 5, 6, 7]]], dtype=torch.float32)
```

i.e., with correct spacing and less of a hack to make it floating point?